src/views/Admin/Pages/Builder.vue

```diff
@@ -314,6 +314,7 @@ export default {
           const { namespaceID } = this.namespace
           this.findPageByID({ namespaceID, pageID: this.pageID, force: true })
             .then(page => {
+              document.title = `${page.title} | ${this.namespace.name} | ${this.$t('general:label.app-name')}`
```

Comment:
Since this is in Admin it should also have `- Admin`
Also this will bug out if page.title or namespace.name are undefined. So use combination of filter and join. Thats the best approach anyway and most durable